gui/src/controls/textBlock.ts

```diff
@@ -81,6 +81,14 @@ module BABYLON.GUI {
             this._markAsDirty();
         }
 
+        public set lineSpacing(value: number) {
+            this._lineSpacing = value;
+        }
+
+        public get lineSpacing(): number {
+            return this._lineSpacing;

```

Comment:
You need to call markAsDirty to force render. But make sure the value actually changed

Reply:
Also this has to be an UnitOrValue object to allow users to use pixel or percentage

Reply:
Cool @deltakosh, thanks for the tips :D.